### ISSUE TITLE
Remove most of the store functions in __utils__

### ISSUE THREAD

**Issue by telekosmos** [OpenPathfinder/visionBoard#158]
We observed there are quite a few functions in `__utils__/index.js` which have the same functionality as those ones in `store` but with slightly different signature. They are only used in tests. So it can be worth to have a look to remove most of them to avoid duplicated code. 

There will have to do changes in some other places (mostly in tests I guess but I couldn't check deeper yet to see if it's worth to go on) to update calls to the function signatures in `store`

**Comment by UlisesGascon** [comment]
Sounds like a plan! We can include the store as a dependency in `__utils__` and simplify a lot of things there. Ideally, we won't extend the `store` to support testing activities—things like `resetDatabase` should remain only in `__utils__`. However, functions like `getAllProjects` can be imported and then exported from the `store`.

**Comment by telekosmos** [comment]
> Sounds like a plan! We can include the store as a dependency in `__utils__` and simplify a lot of things there. Ideally, we won't extend the `store` to support testing activities—things like `resetDatabase` should remain only in `__utils__`. However, functions like `getAllProjects` can be imported and then exported from the `store`.

Good idea about importing then exporting stuff in `store` through `__utils__` 👍
